tests/test_render.py

```diff
@@ -475,6 +475,32 @@ def test_initial_state(self):
 
         assert_clean_console(self)
 
+    def test_array_of_nully_child(self):
+        app = Dash(__name__)
+        app.layout = html.Div(id='nully-wrapper', children=[0])
+
+        self.startServer(app)
+
+        self.wait_for_element_by_css_selector('#nully-wrapper')
+        wrapper = self.driver.find_element_by_id('nully-wrapper')
+
+        self.assertEqual(wrapper.get_attribute('innerHTML'), '0')
```

Comment:
Can't this whole section just be [`self.wait_for_text_to_equal('#nully-wrapper', '0')`](https://github.com/plotly/dash-renderer/blob/master/tests/test_render.py#L59)?

Reply:
Hum.. wasn't aware of that one. Looks like `wait_for_text_to_equal` uses `value` and that would mean it's only meant to be used with `<input />` elements.

Reply:
`value` _or_ `.text`:
https://github.com/plotly/dash-renderer/blob/2c0169c52f5d604db48285814836fe394a0aea07/tests/test_render.py#L63

Reply:
I can't read code it seems. Will update to use suggested.